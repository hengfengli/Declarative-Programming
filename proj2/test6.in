+-----+-----+-----+-----+-----+
|HA...|7.1M.|E4.N2|..D..|.9368|
|9....|.J.N8|6..H.|142PC|.GK.B|
|56...|LB.EP|..1..|..MIA|...C.|
|.OI.4|5.AH2|7.9M.|...K6|.DLF.|
|M2L8N|D6...|GCIAF|...O.|..H.E|
+-----+-----+-----+-----+-----+
|A...J|...B.|FG8.4|P96..|..I..|
|...K9|.15.D|M.C.J|3.IH7|.2...|
|....H|O.76J|.A...|KM...|3584.|
|B...C|G.H.I|.3N.6|OJF48|MEAP9|
|2L.6.|4.8.M|...P.|A..DN|BF.O.|
+-----+-----+-----+-----+-----+
|L.8NA|9.J.E|1.BFG|2CP..|HI.D.|
|J45O1|.M...|LH7DC|6EA..|G3...|
|I.39D|....H|..P.N|7FK.4|OM.B2|
|G..7.|I..1K|5M.93|B.N..|C....|
|.....|6OD.F|4.EIA|M8.3L|N..7.|
+-----+-----+-----+-----+-----+
|3....|...P.|ONFB.|.A5.J|17D9I|
|PNC57|8.M..|2I...|FOE6.|4L.J.|
|D9A..|K24I5|HP...|.G3BM|...8F|
|F.14.|.H..9|.5.6.|..C.2|KPMA.|
|..6EG|JAB..|3L...|...1.|5O.NH|
+-----+-----+-----+-----+-----+
|...J.|.PF..|N.3.7|G....|..5.K|
|7HNFL|286.B|.DM.I|C5...|....J|
|.3...|19O.4|B.2..|.K...|.NPED|
|ED..I|.7N.L|CF6.K|J2OA1|..G.M|
|KB.15|EI...|A..8O|LD7N.|.C4..|
+-----+-----+-----+-----+-----+
