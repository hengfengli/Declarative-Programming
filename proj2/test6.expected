+-----+-----+-----+-----+-----+
|HAJBP|7C1MO|E4KN2|5LDFG|I9368|
|97D3E|FJIN8|6OLH5|142PC|AGKMB|
|56KGF|LB9EP|J813D|HNMIA|247CO|
|1OIC4|5GAH2|7B9MP|E38K6|JDLFN|
|M2L8N|D6K43|GCIAF|97JOB|P1H5E|
+-----+-----+-----+-----+-----+
|A1OMJ|3NEBC|FG8L4|P9625|DKIH7|
|48FK9|P15AD|MECOJ|3BIH7|62NGL|
|NGPIH|OF76J|9AD2B|KMLCE|35841|
|B57DC|GLH2I|K3N16|OJF48|MEAP9|
|2LE63|4K89M|I75PH|A1GDN|BFJOC|
+-----+-----+-----+-----+-----+
|LM8NA|93J7E|1KBFG|2CP5O|HI6D4|
|J45O1|BM28N|LH7DC|6EA9I|G3FKP|
|IE39D|A5CLH|86PJN|7FKG4|OM1B2|
|GF276|I4P1K|5MO93|BHNJD|C8ELA|
|CPBHK|6ODGF|42EIA|M813L|NJ975|
+-----+-----+-----+-----+-----+
|3KH2M|CEGP6|ONFB8|4A5LJ|17D9I|
|PNC57|8DM31|2IAK9|FOE6H|4LBJG|
|D9ALO|K24I5|HPJ71|NG3BM|E6C8F|
|FJ14B|NHLO9|D5G6E|8IC72|KPMA3|
|8I6EG|JABF7|3L4CM|DP91K|5O2NH|
+-----+-----+-----+-----+-----+
|OC4J2|MPFDA|N13E7|G6B89|LH5IK|
|7HNFL|286KB|PDMGI|C54E3|9AO1J|
|63GA8|19OC4|BJ25L|IKHMF|7NPED|
|ED9PI|H7N5L|CF64K|J2OA1|8BG3M|
|KBM15|EI3JG|A9H8O|LD7NP|FC426|
+-----+-----+-----+-----+-----+
